qiita_db/support_files/patches/42.sql

```diff
@@ -0,0 +1,15 @@
+-- Nov 19, 2016
+-- Adding provenance parameter to validate commands
+
+DO $do$
+DECLARE
+    cmd  RECORD;
+BEGIN
+    FOR cmd IN
+        SELECT command_id FROM qiita.software_command WHERE name = 'Validate'
+    LOOP
+        INSERT INTO qiita.command_parameter (command_id, parameter_name, parameter_type, required, default_value)
```

Comment:
Got it, thanks. Just to confirm, this value is not required or defined by the artifacts, they are defined as part of the Validate command that generated those artifacts, right?